benchmarking/runs/run_diversify_social_only.py

```diff
@@ -0,0 +1,90 @@
+import math
+
+from benchmarking.data.interfaces import MockStudentProviderSettings
+from benchmarking.data.simulated_data.mock_student_provider import MockStudentProvider
+from benchmarking.evaluations.metrics.average_social_satisfied import (
+    AverageSocialSatisfaction,
+)
+from benchmarking.evaluations.metrics.num_requirements_satisfied import (
+    NumRequirementsSatisfied,
+)
+from benchmarking.evaluations.metrics.num_teams_meeting_requirements import (
+    NumTeamsMeetingRequirements,
+)
+from benchmarking.evaluations.metrics.utils.team_option import TeamOption
+from benchmarking.evaluations.scenarios.diversify_social_only import (
+    DiversifySocialFriends,
+)
+from benchmarking.simulation.simulation import Simulation
+from models.enums import ScenarioAttribute, Gender
+
+
+def run_diversify_2_friends():
```

Comment:
Same comments as with naming in the scenario applies to this function and file's name

I don't think the word "diversify" makes sense here